src/components/close-button/__tests__/CloseButton.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CloseButton from '../CloseButton';
+import Button from '../../button';
+
+describe('components/close-button/CloseButton', () => {
+    test('should correctly render', () => {
+        const wrapper = shallow(<CloseButton />);
+
+        expect(wrapper.find(Button).length).toEqual(1);
+        expect(wrapper.hasClass('bdl-CloseButton')).toBe(true);
+        expect(wrapper).toMatchSnapshot();
```

Comment:
We generally want to avoid snapshot testing. Do we need to add some more explicit assertions, instead?